### PR TITLE
First cut at adding documentation for accessing the cluster

### PR DESCRIPTION
**Pull request by satnam6502**
This is a first cut at addressing  #3996

### CODE REVIEW

File: docs/accessing_the_cluster.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Accessing the Cluster
+
+## Using the Kubernetes proxy to access the cluster
+Information about the cluster can be accessed by using a proxy URL and by providing the keys to the cluster.
+For example, for a cluster that has cluster level logging enabled using Elasticsearch you can fetch information about
```
Comment (bgrant0607) [inline docs/accessing_the_cluster.md]: cluster-level

Comment (satnam6502) [inline docs/accessing_the_cluster.md]: Done.

File: docs/accessing_the_cluster.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  "User": "admin",
+  "Password": "4mty0Vl9nNFfwLJz",
+  "CAFile": "/Users/satnam/.kube/kubernetes-satnam2_kubernetes/kubernetes.ca.crt",
+  "CertFile": "/Users/satnam/.kube/kubernetes-satnam2_kubernetes/kubecfg.crt",
+  "KeyFile": "/Users/satnam/.kube/kubernetes-satnam2_kubernetes/kubecfg.key"
+}
+```
+
+To access a service endpoint `/A/B/C` via the proxy service for your service S you need to specify an HTTPS address
+for the Kubernetes master followed by `/api/v1beta1/proxy/S/A/B/C/`. Currently it is important to
```
Comment (bgrant0607) [inline docs/accessing_the_cluster.md]: This is confusing. `S` needs to expand into a path.

Comment (bgrant0607) [inline docs/accessing_the_cluster.md]: Please include the namespace in the path, so users can tell where it should go. Using the default namespace is fine.

Comment (bgrant0607) [inline docs/accessing_the_cluster.md]: Nevermind about namespace. It doesn't work the way I thought it did in v1beta1.

Comment (satnam6502) [inline docs/accessing_the_cluster.md]: Added missing `service`, slightly de-tersed.

**Comment by satnam6502** [comment]
Changed filename from using _ to - as separator.

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**